src/constants/landscape-view-constants.js

```diff
@@ -1 +1,7 @@
-export const ZOOM_LEVEL_TRIGGER = 8;
\ No newline at end of file
+export const ZOOM_LEVEL_TRIGGER = 8;
+
+export const LAYER_VARIANTS = {
```

Comment:
This `landscape-view-constants` is storing ``landscape view related data. Maybe it makes more sense to store it inside another dedicated to biodiversity layers?